ikarus/solver/nonlinearsolver/newtonraphson.hh

```diff
@@ -131,7 +132,12 @@ public:
    * \brief Set up the solver with the given settings.
    * \param settings Newton-Raphson settings.
    */
-  void setup(const Settings& settings) { settings_ = settings; }
+  void setup(const Settings& settings) {
+    if (settings.minIter > settings.maxIter)
+      DUNE_THROW(Dune::InvalidStateException,
```

Comment:
We can check this with a checkThrow somewhere?

Reply:
Ah sry I only saw the codecov report saying that this line is not tested, then nvm